src/App.js

```diff
@@ -9,7 +10,11 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <>
+    <Article title={article.title} />
+    <Article text={article.text} />
+    <Article date={article.date} />
+  </>
```

Comment:
```suggestion
    <Article { ...article } />
```
There's should be only one component `Article` which receives the object with `title, date, text` fields and renders them inside